scripts/lib/Utils.psm1

```diff
@@ -238,6 +238,7 @@ Function registerTest($testname, $index, $bucket, $filter, $moreParams, $cluster
         if ($testname -eq "agency")
         {
             $dumpAgencyOnError = "true"
+            $testWeight = 2
```

Comment:
I'd rather add `-weight 2` here:
https://github.com/arangodb/oskar/blob/master/scripts/runTests.ps1#L41

Otherwise  it will alter 4 weight for cluster agency test.